src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs

```diff
@@ -727,16 +719,20 @@ private void ItemSelected_Unchecked(object sender, RoutedEventArgs e)
 
 		private new void FileList_ContainerContentChanging(ListViewBase sender, ContainerContentChangingEventArgs args)
 		{
-			args.ItemContainer.PointerEntered -= ItemRow_PointerEntered;
-			args.ItemContainer.PointerExited -= ItemRow_PointerExited;
-			args.ItemContainer.PointerCanceled -= ItemRow_PointerCanceled;
+			args.ItemContainer.FindDescendant("IconBox")!.PointerEntered -= IconBox_PointerEntered;
+			args.ItemContainer.FindDescendant("IconBox")!.PointerExited -= IconBox_PointerExited;
+			args.ItemContainer.FindDescendant("IconBox")!.PointerCanceled -= IconBox_PointerCanceled;
+			args.ItemContainer.FindDescendant("SelectionCheckbox")!.PointerExited -= SelectionCheckbox_PointerExited;
+			args.ItemContainer.FindDescendant("SelectionCheckbox")!.PointerCanceled -= SelectionCheckbox_PointerCanceled;
```

Comment:
I think it would be better if we use `.FindDescendant("IconBox")` and `.FindDescendant("SelectionCheckbox")` only once. I mean storing their value in a `var` and then accessing it

Reply:
Certainly. I fixed.